### ISSUE TITLE
Empty 'logs' directory is getting created under 's3-config' directory 

### ISSUE THREAD

**Issue by anandhu-karattu** [noobaa/noobaa-core#7930]
### Environment info
- NooBaa Version: **noobaa-core-5.15.0-20240320.el9.x86_64 5.15 d/s**
- Platform: **Standalone Nobaa**

### Actual behavior
An empty 'logs' directory is created under s3-config directory after executing "bucket add" command.

```
[root@mars-x-worker1 s3-config]# pwd
/gpfs/fs-ces-shared/ces/s3-config
[root@mars-x-worker1 s3-config]# ls -lrt
total 3
-rwx------ 1 root root  377 Mar 28 03:00 config.json
-rw-r--r-- 1 root root  209 Mar 28 03:01 system.json
drwx------ 2 root root 4096 Mar 28 03:03 accounts
drwx------ 2 root root 4096 Mar 28 03:03 access_keys
drwx------ 2 root root 4096 Mar 28 03:06 buckets
drwxr-xr-x 3 root root 4096 Mar 28 03:07 logs
[root@mars-x-worker1 s3-config]# tree logs
logs
└── logs

1 directory, 0 files
```

### Expected behavior
This 'logs' directory is not expected under s3-config directory.
Since we are taking periodic backup of s3-config directory in scale, we are taking backup of this un necessary log file.(we are intended to take only config backup)

### Steps to reproduce
1. Create a bucket via node bucket create command.

**Comment by anandhu-karattu** [comment]
In fact the 'logs' directory is getting created inside the directory where the "bucket add" command is being called. 
```
For eg:
[root@mars-x-worker1 s3-config]# ls
access_keys  accounts  buckets  config.json  system.json
[root@mars-x-worker1 s3-config]# /usr/local/noobaa-core/bin/node /usr/local/noobaa-core/src/cmd/manage_nsfs bucket add --name buck9999 --owner anandhu9999 --path /mnt/fs1/anandhu9999/buck9999 2>/dev/null
{
  "response": {
    "code": "BucketCreated",
    "reply": {
      "_id": "6605474ca194de41f7dc5127",
      "name": "buck9999",
      "owner_account": "6605407d69f372d14696b40f",
      "system_owner": "anandhu9999",
      "bucket_owner": "anandhu9999",
      "versioning": "DISABLED",
      "creation_date": "2024-03-28T10:32:44.636Z",
      "path": "/mnt/fs1/anandhu9999/buck9999",
      "should_create_underlying_storage": false,
      "fs_backend": "GPFS"
    }
  }
}
[root@mars-x-worker1 s3-config]# ls
access_keys  accounts  buckets  config.json  logs  system.json
```

**Comment by ramya-c3** [comment]
It is not only creating for bucket create infact logs dummy folder is creating for all the node commands

**Comment by naveenpaul1** [comment]
`logs` folder is getting created in folder location from where the manage nsfs node command is executed.
If manage nsfs command run from `tmp` folder `logs` folder will create in `tmp` folder
@anandhu-karattu

**Comment by ramya-c3** [comment]
@naveenpaul1 Why we are creating empty log folders without content from whereever we are executing the command?

**Comment by romayalon** [comment]
@anandhu-karattu @naveenpaul1 Adding request validation label

**Comment by anandhu-karattu** [comment]
Picking this up for verification in latest u/s build @naveenpaul1

**Comment by anandhu-karattu** [comment]
Verified on upstream build "noobaa-core-5.17.0-20240415.el9.x86_64"
logs directory is no more getting created. 
For eg:
```
[root@earth-x-worker1 s3-config]# /usr/local/noobaa-core/bin/node /usr/local/noobaa-core/src/cmd/manage_nsfs bucket add --name bucket1000 --owner account-35940 --path /mnt/fs1/bucket_35940/bucket100 2>/dev/null
{
  "response": {
    "code": "BucketCreated",
    "reply": {
      "_id": "661f95aed3a8bf3d7e901d3b",
      "name": "bucket1000",
      "owner_account": "661f772ef7cb9fc8c3e2b1e7",
      "system_owner": "account-35940",
      "bucket_owner": "account-35940",
      "versioning": "DISABLED",
      "creation_date": "2024-04-17T09:26:06.623Z",
      "path": "/mnt/fs1/bucket_35940/bucket100",
      "should_create_underlying_storage": false,
      "fs_backend": "GPFS"
    }
  }
}
[root@earth-x-worker1 s3-config]# ls
access_keys  accounts  buckets  config.json  master_keys.json  system.json
```
Also verified under '/root', there is no logs directory getting created.
Fix looks good.

**Comment by anandhu-karattu** [comment]
@nimrod-becker we will need this fix back ported to 4.15.x. Can you please help

**Comment by nimrod-becker** [comment]
It was already backported to the 5.15 branch
